src/SFA.DAS.AANHub.Application/SFA.DAS.AANHub.Application.csproj

```diff
@@ -18,4 +18,8 @@
   <ItemGroup>
     <ProjectReference Include="..\SFA.DAS.AANHub.Domain\SFA.DAS.AANHub.Domain.csproj" />
   </ItemGroup>
+
+  <ItemGroup>
```

Comment:
Another folder has crept in, can be removed

Reply:
Thanks for spotting Liam. 
Removed now. Could be getting added in due to the way I do the "open in explorer view". I have to keep this under observation.